homeassistant/components/prometheus/__init__.py

```diff
@@ -54,12 +54,14 @@
     {vol.Optional(CONF_OVERRIDE_METRIC): cv.string}
 )
 
+DEFAULT_NAMESPACE = "hass"
```

Comment:
Is `home-assistant` or `homeassistant` too long?

Reply:
The minus symbol (`-`) is [not allowed](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels) in Prometheus metric names, but `homeassistant` is OK (and is actually what I am using personally). However, our [configuration example](https://www.home-assistant.io/integrations/prometheus/#full-example) uses `hass`, so I've used it here as well. Would you recommend `homeassistant` instead?

Reply:
Yes, we prefer using `homeassistant` instead of `hass`.

Reply:
I've changed the default namespace to `homeassistant`.